src/graphics/PlanningView.ts

```diff
@@ -606,51 +605,43 @@ export async function runPredictions(underworld: Underworld) {
         }
       }
 
-      // Run onTurnStartEvents on unitsPrediction:
-      // Displays markers above units heads if they will attack the current client's unit
-      // next turn
+      const aiUnits = underworld.unitsPrediction.filter(u => u.unitType == UnitType.AI);
+
+      // Careful when making enemy predictions:
+      // Ally turn happens before Enemy turn, and some effects
+      // like bloat may not be easy to factor into predictions
+
+      // TODO - Run turn start events for units that will run it?
```

Comment:
Is there an issue in github tracking this?

Reply:
If so, please reference in the TODO comment

Reply:
Sort've. There is #409 , but it is just one part of this larger issue, which affects predictions, smart targeting, etc.

I will look into making a more generic issue and referencing it in the TODO, and also attach related issues to it, like I did with #362

Reply:
I more meant, if we're going to leave TODO's in the codebase, let's include a comment with it in the code that references the same TODO tracked in Github issues so that if we come across it later in the code base we can go to the link and see any conversation or current state of the issue.  I'd like that to be Best Practice when leaving TODO's in code comments.

Reply:
No I know, I linked the issue in the comment with the last set of changes. Just wanted to provide additional context/thoughts here

Reply:
Ah this was due to me looking at an old commit in this PR. Whoops!